Remotion/Core/Reflection.UnitTests/CodeGeneration/MethodWrapperEmitterTests/SetUpFixture.cs

```diff
@@ -46,13 +46,20 @@ public virtual void OneTimeSetUp ()
       Console.WriteLine ("Setting up MethodWrapperEmitterTests");
 
       var assemblyName = new AssemblyName ("Remotion.Reflection.CodeGeneration.MethodWrapperEmitterTests.Generated.Unsigned");
+#if !NO_PEVERIFY
       s_assemblyBuilder = AppDomain.CurrentDomain.DefineDynamicAssembly (
-          assemblyName,
-          AssemblyBuilderAccess.RunAndSave,
-          TestContext.CurrentContext.TestDirectory);
-
+                assemblyName,
+                AssemblyBuilderAccess.RunAndSave,
+                TestContext.CurrentContext.TestDirectory);
+#else
+      s_assemblyBuilder = AssemblyBuilder.DefineDynamicAssembly (assemblyName, AssemblyBuilderAccess.Run);
+#endif
       var moduleName = assemblyName + ".dll";
```

Comment:
Maybe it would be best to move this line to line 48 and then we can use a single IF-def for both code blocks.